syft/core/hooks/torch/hook.py

```diff
@@ -585,28 +462,17 @@ def _hook_tensor(self, tensor_type):
                     setattr(tensor_type, attr, new_attr)
 
         # Add in our own Grid-specific methods
-        self._hook_tensor_send_(tensor_type)
+        self._hook_send_(tensor_type)
         self._hook_get_(tensor_type)
         self._hook_tensor__ser(tensor_type)
 
-    def _hook_tensor___init__(hook_self, tensor_type):
-        """Overload tensor_type.__init__"""
-
-        def new___init__(self, *args):
-            super(tensor_type, self).__init__()
-
-            # self = hook_self.local_worker.register_object(hook_self.local_worker,
-            # self, is_pointer=False)
-
-        tensor_type.__init__ = new___init__
-
     def _hook_tensor___del__(hook_self, tensor_type):
         def new____del__(self, *args):
             print("deleting tensor")
 
         tensor_type.__del__ = new____del__
 
-    def _hook_tensor___new__(hook_self, tensor_type):
+    def _hook___new__(hook_self, tensor_type):
```

Comment:
Is this now generalized to include Variables and Parameters?  If so, the docstring and argument `tensor_type` need to be modified/renamed.

Reply:
Indeed it is. Good catch on docstring.

Reply:
is tensorvar_type a reasonable replacement variable name? seems like that's the convention we've been going for

Reply:
torch_type is a little nicer to look at but tensorvar_type is a bit more specific

Reply:
Yeah - in the long run i'd imagine we'll be doing all sorts of stuff with torch types - tensorvar might last a bit longer